src/Uno.Extensions.Maui.UI/MauiContentCreatedEventArgs.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Uno.Extensions.Maui;
+
+public class MauiContentCreatedEventArgs : EventArgs
+{
+	public MauiContentCreatedEventArgs(VisualElement content)
+	{
+		Content = content;
+	}
+
+	public VisualElement Content { get; }
```

Comment:
Is this class necessary, over just an eventargs? You still have access to the maui view from the sender.

Reply:
@dotMorten the idea was to make it easier to access the VisualElement without having to cast the sender. Is there a reason why you'd prefer just an EventArgs?

Reply:
Just a slimmer api. No bigger reason than that